tests/test_journal.py

```diff
@@ -46,8 +46,59 @@ def entry(db_session):
 # all the tests using the app fixture
 
 
+def test_permalink_exists(db_session, app):
+    """Tests that a new view is generated when a new entry is
+    added and that the new view has a permalink based on the
+    title of the new entry.
+    """
+    data = {'text': 'test text', 'title': "Title"}
+    journal.Entry.write(session=db_session, **data)
+    db_session.flush()
+    response = app.get('/8/Title')
```

Comment:
Are you certain that the ID of the post you just wrote is always '8' when you are testing?  Why hard-code it?  Can you get it from the entry you just wrote?